tuxemon/core/save.py

```diff
@@ -189,3 +190,11 @@ def get_index_of_latest_save():
     else:
         return None
 
+
+def get_save_path(slot):
+    # append the mod names if this is a custom game	
+    mods = ""
+    if len(prepare.CONFIG.mods) > 1:
+        for i in range(0, len(prepare.CONFIG.mods) - 1):
+            mods += prepare.CONFIG.mods[i] + "-"
```

Comment:
If prepare.CONFIG.mods = ["a", "b", "c"]
then 
mods = "a-b-"

Did you want "a-b-c"?

Just do "-".join(prepare.CONFIG.mods)

Reply:
To get the trailing "-", change the format string instead:

 '{}-slot{}.save'

Reply:
If you really want to miss out the last mod then do "-".join(prepare.CONFIG.mods[:-1])

Reply:
I try to simplify the code as much as possible. It's to ensure that the default mod at the end of the array (tuxemon) is ignored since it should be constant, but all others are added.

For instance: Let's say the mods array is just `["tuxemon"]` so no custom game. The loop won't execute then, so the mods variable remains "". When used you get just "slot1.save".

Now let's say there is a second mod called game so we have `["game", "tuxemon"]`. The loop will run once, the mods variable now becomes "game-". When appended before the save name the full name becomes "game-slot1.save".

In case you're using a mixture of two mods (full support might come later on): `["game1", "game2", "tuxemon"]`. The loop now runs twice and sets the mods prefix to "game1-" then "game1-game2-", to which the save name is then added to get "game1-game2-slot1.save".

Reply:
If you believe supporting multiple mod combinations is pointless, I can simply use `prepare.CONFIG.mods[0]` to append just the name of the first mod being used. But if we have mod dependencies later on, saves might break if you run a mod without other required mods.

Let me know if you still consider the current format bad and think I should go this route. For now it would do fine in my book... the important part is to at least customize saves per one mod.

Reply:
As for the trailing "-": I append it per mod so that it's not added when no mod is used but separates multiple mods, as described above. Without that, in case there is no mod, we'd get `-slot1.save`. Meanwhile if there are multiple mods such as "game1" and "game2", appending it only once would result in `game1game2-slot1.save` (no separator between the two names).

Reply:
Cool, I'd probably go for this:

```
# The last element is always "tuxemon", ignore it
mods = prepare.CONFIG.mods[:-1]
if mods:
    mod_prefix = "-".join(mods) + "-"
else:
    mod_prefix = ""
return os.path.join(prepare.SAVE_PATH, '{}slot{}.save'.format(mod_prefix, slot))
```

Reply:
That's definitely cleaner and more efficient! Thanks for the suggestion: Just tested and committed, works very good.